src/fields/components/Placeholder.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import styled from 'styled-components'
+
+import { Box } from '../../Box'
+import { theme } from '../../theme'
+
+export const Placeholder = () => {
```

Comment:
<img width="494" alt="Screenshot 2022-09-01 at 11 34 00" src="https://user-images.githubusercontent.com/14129033/187894181-47b07bdc-1064-48fb-8495-f8342d35683b.png">

Placeholder component for stories